client/src/partials/reports_proposed/variation_exploitation/variation_exploitation.js

```diff
@@ -0,0 +1,69 @@
+var variationExploitationController = function ($http, $translate, validate, reportConfigService, messenger) {
+  var dependencies = {}, generatedDocumentPath = null, serverUtilityPath = 'report/build/variation_exploitation', 
+    configuration = reportConfigService.configuration, vm = this;
+
+  dependencies.fiscalYears = {
+    query : {
+      identifier : 'id',
+      tables : {
+        'fiscal_year' : {
+          columns : ['id', 'fiscal_year_txt', 'previous_fiscal_year']
+        }
+      }
+    }
+  };
+
+  vm.generate_doc = $translate.instant('VARIATION_EXPLOITATION.GENERATE_DOC');
+  vm.loading = $translate.instant('VARIATION_EXPLOITATION.LOADING');
+  vm.building = false;
+  vm.configuration = configuration;
```

Comment:
I like to expose comtroller function to the scope at the bottom and initialize some value at the top but I will put them at the bottom.